Cargo.toml

```diff
@@ -32,7 +32,7 @@ bytes = "0.4"
 crossbeam-channel = "0.3"
 curl = "^0.4.20"
 curl-sys = "^0.4.20"
-futures-io-preview = "=0.3.0-alpha.17"
+futures-io-preview = "=0.3.0-alpha.18"
```

Comment:
Is '=' version binding necessary?

Reply:
It was once upon a time, but it should be safe to remove now. There used to be some weird runtime issues with a couple older alphas, but that has been resolved since.